shoop/campaigns/forms.py

```diff
@@ -114,13 +130,19 @@ def _get_many_to_many_initial_values(self, obj):
             return [initial_value]
 
     def clean(self):
-        """ Ensure at least one rule is set """
         data = super(CampaignFormMixin, self).clean()
-        if data.get("discount_percentage") and data.get("discount_amount_value"):
-            raise ValidationError(_("You should only define either discount percentage or amount."))
 
-        if not any([data.get("discount_percentage"), data.get("discount_amount_value")]):
-            raise ValidationError(_("You must define discount percentage or amount."))
+        # Ensure at least 1 effect is set
+        effect_count = 0
+        for key in self.effect_keys:
+            for cls in get_provide_objects(key):
+                if data[cls.identifier]:
+                    effect_count += 1
+
+                if effect_count > 1:
+                    raise ValidationError(_("You should only define one effect."))
+        if not effect_count:
+            raise ValidationError(_("At least one effect must be defined."))
```

Comment:
yes, this is the very reason. We start by having one (to mimic current features) and move forward with multiple in future.